charts/README.md

```diff
@@ -39,33 +39,34 @@ To know more about the various flag options used with upgrade command check out
+| `image.registry`                                   | Image registry for cosi-driver-nutanix sidecar                             | `ghcr.io/`                                                                   |
+| `image.repository`                                 | Image repository for cosi-driver-nutanix sidecar                           | `nutanix-cloud-native/cosi-driver-nutanix`                                   |
+| `image.tag`                                        | Image tag for cosi-driver-nutanix sidecar                                  | `""`                                                                         |
+| `image.pullPolicy`                                 | Image registry for cosi-driver-nutanix sidecar                             | `IfNotPresent`                                                               |
+| `secret.enabled`                                   | Enables K8s secret deployment for Nutanix Object Store                     | `true`                                                                       |
+| `secret.endpoint`                                  | Nutanix Object Store instance endpoint                                     | `""`                                                                         |
+| `secret.access_key`                                | Admin IAM Access key to be used for Nutanix Objects                        | `""`                                                                         |
+| `secret.secret_key`                                | Admin IAM Secret key to be used for Nutanix Objects                        | `""`                                                                         |
+| `secret.pc_ip`                                     | PC ip                                                                      | `""`                                                                         |
+| `secret.pc_port`                                   | PC port                                                                    | `""`                                                                         |
+| `secret.pc_username`                               | PC username                                                                | `""`                                                                         |
+| `secret.pc_password`                               | PC password                                                                | `""`                                                                         |
+| `secret.account_name`                              | Account Name is a displayName identifier Prefix for Nutanix                | `"ntnx-cosi-iam-user"`                                                       |
+| `cosiController.create`                            | Whether to create the COSI central controller deployment and its resources | `true`                                                                       |
```

Comment:
can we just change create by enable, thanks

Reply:
Yep done in https://github.com/nutanix-cloud-native/cosi-driver-nutanix/pull/375/commits/caf508b1df163566d0692c0de7e29349a34b5b3f, thanks